lib/Launcher.js

```diff
@@ -83,26 +83,32 @@ class Launcher {
 
     // Cleanup as processes exit.
     const listeners = [
-      helper.addEventListener(process, 'exit', killChromeAndCleanup),
-      helper.addEventListener(chromeProcess, 'exit', killChromeAndCleanup),
+      helper.addEventListener(process, 'exit', killChrome),
```

Comment:
once doesn't solve it completely because there are multiple paths into `killChrome`. I added a flag.